soc/arm/nordic_nrf/nrf54l/Kconfig.defconfig.nrf54l15_enga_cpuapp

```diff
@@ -0,0 +1,23 @@
+# Nordic Semiconductor nRF54L15 MCU
+
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_NRF54L15_ENGA_CPUAPP
+
+config SOC
+	string
+	default "nRF54L15_CPUAPP"
+
+config SOC_DOMAIN_NAME
+	string
+	default "application"
```

Comment:
this is not used anywhere, drop, also this is likely redundant (see above, you can deduce you're in app)

Reply:
Removed